presage/src/manager.rs

```diff
@@ -953,6 +959,65 @@ impl<C: Store> Manager<C, Registered> {
                                         log::trace!("{group:?}");
                                     }
                                 }
+                                let uuid = content.metadata.sender.uuid.clone();
+                                match state.config_store.contact_by_id(uuid) {
```

Comment:
I feel like the feature in itself is a good addition, but I would maybe do this with a function like `upsert_contact` similar to `upsert_group`.